src/parser/core/modules/Medicated.ts

```diff
@@ -0,0 +1,71 @@
+import {Events} from 'event'
+import {Analyser} from 'parser/core/Analyser'
+import {dependency} from '../Injectable'
+import {Data} from './Data'
+import {SimpleRow, StatusItem, Timeline} from './Timeline'
+
+interface Medication {
+	start: number
+	end?: number
+}
+
+export class Medicated extends Analyser {
+	static handle = 'medicated'
+
+	@dependency private data!: Data
+	@dependency private timeline!: Timeline
+
+	private pot: Medication | undefined
+	private pots: Medication[] = []
+
+	initialise() {
+		this.addEventHook({
+			type: 'statusApply',
+			target: this.parser.actor.id,
+			status: this.data.statuses.MEDICATED.id,
+		}, this.onMedication)
+
+		this.addEventHook({
+			type: 'statusRemove',
+			target: this.parser.actor.id,
+			status: this.data.statuses.MEDICATED.id,
+		}, this.offMedication)
+
+		this.addEventHook('complete', this.onComplete)
+	}
+
+	private onMedication(event: Events['statusApply']) {
+		this.pot = {start: event.timestamp - this.parser.pull.timestamp}
```

Comment:
I'd avoid storing the data in timeline-specific format. Preference storing event timestamps, and adjusting for timeline display values when actually adding to the timeline. Timeline will eventually be adjusted to handle proper event timestamps.